app/controllers/qualifications_interface_controller.rb

```diff
@@ -0,0 +1,13 @@
+class QualificationsInterfaceController < ApplicationController
+  before_action :authenticate_user!
+  before_action :handle_expired_token!
+
+  def handle_expired_token!
+    redirect_to sign_out_path unless session[:identity_user_token_expiry]
+
+    if Time.zone.now > Time.zone.at(session[:identity_user_token_expiry])
```

Comment:
Just a nit picking suggestion here. I like to use the ActiveSupport helpers to make code like this more readable. `>` and `<` always confuse me.

```suggestion
    if Time.zone.at(session[:identity_user_token_expiry]).past?
```